docs/source/guide/custom_metric.md

```diff
@@ -11,6 +11,8 @@ section: "Quality control"
 
 ---
 
+<div class="enterprise-only">
```

Comment:
@KonstantinKorotaev  This whole page has already moved to enterprise, please remove this div out from here.

Reply:
![image](https://user-images.githubusercontent.com/501780/208798053-a7313d18-5176-426c-ba23-67f16a32ea54.png)

Reply:
Fixed